packages/twenty-server/src/engine/core-modules/tool/tools/send-email-tool/send-email-tool.ts

```diff
@@ -115,17 +118,23 @@ export class SendEmailTool implements Tool {
         workspaceId,
       );
 
+      const parsedBody = parseEmailBody(body);
+      const reactMarkup = reactMarkupFromJSON(parsedBody);
+      const htmlBody = await render(reactMarkup);
+      const textBody = toPlainText(htmlBody);
+
       const { JSDOM } = await import('jsdom');
       const window = new JSDOM('').window;
       const purify = DOMPurify(window);
-      const safeBody = purify.sanitize(body || '');
+      const safeHtmlBody = purify.sanitize(htmlBody || '');
       const safeSubject = purify.sanitize(subject || '');
 
```

Comment:
You had mentioned an issue with one of the 3 providers (smtp/microsft/gmail), is it something we should be concerned before merge?